script/gen_requirements_all.py

```diff
@@ -24,6 +24,7 @@
     "beacontools",
     "beewi-smartclim",  # depends on bluepy
     "bluepy",
+    "dacite",
```

Comment:
Why add it to comment requirements?

Reply:
To be honest, I thought comment requirements just meant that it had a comment further down below. I see now that I misunderstood that. Removed.